airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -48,7 +48,7 @@ class VaultBackend(BaseSecretsBackend, LoggingMixin):
         (default: 'connections').
     :type connections_path: str
     :param variables_path: Specifies the path of the secret to read to get Variables
```

Comment:
Can you describe the meaning of the special value "None"?

Reply:
Is something like this ok?

```
    :param connections_path: Specifies the path of the secret to read to get Connections
        If set to None, requests for connections will not be sent to Vault (default: 'connections').
    :type connections_path: str
    :param variables_path: Specifies the path of the secret to read to get Variables
        If set to None, requests for variables will not be sent to Vault (default: 'variables').
```